.eslintrc.json

```diff
@@ -28,34 +28,34 @@
           // Similar to above but for src/www/model, but you can use files from both the
           // src/www/model and src/www/infrastructure paths.
           {
-            "target": "./src/www/model",
+            "target": "./client/src/wwwmodel",
```

Comment:
Should it be this?
```suggestion
            "target": "./client/src/www/model",
```

Reply:
Whoops yes thanks, don't know what happened here.